nilearn/reporting/glm_reporter.py

```diff
@@ -872,7 +930,10 @@ def _stat_map_to_png(
                 display_mode=display_mode,
                 cmap=cmap,
                 symmetric_cbar=symmetric_cbar,
-                threshold=threshold,
```

Comment:
Instead of removing threshold could we pass `abs(threshold)`. That would help make clear on the plot that those values are thresholded. In the below figures, it is written in the title, but not clear on the images. Also for two_sided thresholded images, the thresholded parts are all white, but not for one sided cases.

Reply:
Sorry, you are right.